config.js

```diff
@@ -4,7 +4,9 @@ var config = {
   twitter: (process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot').toLowerCase().split(','),
 
   // Use this to have a different animal say a message in the big box.
-  say: (process.env.TTC_SAY_BOX || 'parrot').toLowerCase(),
+  // regex: if TTC_SAY_BOX is a filePath, return that path
+  say: /(\w[~\/])/.test(process.env.TTC_SAY_BOX)
```

Comment:
I think the config.js should be as simple as possible.
I would recommend removing the `toLowerCase` from the config.js all together and making the necessesarry checks in [care.js#L316](https://github.com/vinnyA3/tiny-care-terminal/blob/master/care.js#L316)

@vinnyA3 What do you think?